core/src/main/java/org/jdbi/v3/core/interceptor/JdbiInterceptionChainHolder.java

```diff
@@ -34,15 +34,6 @@
 @Alpha
 public final class JdbiInterceptionChainHolder<S, T> {
 
-    @SuppressWarnings("UnnecessaryLambda") // constant for readablity
```

Comment:
I would have preferred that change in a separate PR. It kind of gets lost between the annotation changes.